game-starfield/index.js

```diff
@@ -0,0 +1,17 @@
+function main(context) {
+  context.registerGameStub({
+    id: 'starfield',
+    executable: null,
+    mergeMods: false,
+    name: 'Starfield',
+    queryModPath: () => '',
```

Comment:
It's a stub so it shouldn't matter, but given how Vortex resolves the mod path, this should be changed to `queryModPath: () => '.',`